docs/project/2024/tudor.arhir/index.md

```diff
@@ -0,0 +1,81 @@
+# Breathalyzer
+A breathalyzer which detects the alcohol in the blown air.
+
+:::info 
+
+**Author**: Tudor-Cristian Arhir \
+**GitHub Project Link**: https://github.com/UPB-FILS-MA/project-TudorArhir
+
+:::
+
+## Description
+
+ My project is a breathalyzer with a LCD (which shows the precentage of alchohol in the air(and some other stuff)), a buzzer, an alcohol sensor, 2 LED's  and a button which resets the number so we can have another reading.( the "no idea" price tag on the components is there because they come from an arduino kit which i bought 1 year ago and i cannot remember the price). It has a 3D-printed case which makes it look more practical than just wires and components on a breadboard . I want this project to be something useful and at the same time something fun which anybody can use.
+
+
+## Motivation
+
+I chose this project based on two important factors:
+-on my 20th birthday my friends got me a real breathalyzer which is very usefull for me because i love to drive
+-i wanted to make it more practical and easy to use for anybody at any time(we use it at gatherings and parties)
```

Comment:
```suggestion
 - on my 20th birthday my friends got me a real breathalyzer which is very usefull for me because i love to drive
 - I wanted to make it more practical and easy to use for anybody at any time(we use it at gatherings and parties)
```